### DISCUSSION TITLE
Armbian Bookworm vs Joshua Riek

### DISCUSSION THREAD

**Discussion by Bahazzie** [Joshua-Riek/ubuntu-rockchip#579] [General]
Hello all and Joshua,

Played with Armbian Bookworm  for months and it was great, got RealVNC working and worked headless. Then started messing with overclocking in armbian-config and after this it didn't boot anymore from sata ssd. Also Armbian Bookworm didn't do hardware acceleration. Now I browsed Geekbench 6 and it seems JR Ubunu scores the highest and it also seems to support hardware accel. So I am ready for a new try. Pity I have to figure out how to boot from SATA SSD again, as well as how to get ReallVNC working again. But yeah, you sbc or you not sbc.

(got the alu passively cooled case, it works great and is completely silent, even after stresstesting 8 cores the the max for an hour there is no throttling)

Bas

**Comment by InfernalWolf666** [comment]
убунту господина Джошуа Риека самая лучшая операционная система !

**Reply by Bahazzie** [reply]
Thanks for that, I translated it :-)
